.github/workflows/CI.yml

```diff
@@ -139,7 +140,10 @@ jobs:
       - name: Install commitlint
         run: ./commitlint.sh --version
       - name: Run tests to validate our plugins
-        run: yarn jest --config jest.config.ts
+        run: |
+          # remove *.js files so that the same tests don't run twice
+          rm ./commitlint/tests/*.js
```

Comment:
@webwarrior-ws I believe this is a problem in master branch too so let's put this as a separate commit before this commit

Reply:
Moved to separate commit